src/streaming/text/TextSourceBuffer.js

```diff
@@ -394,6 +392,9 @@ function TextSourceBuffer(config) {
         try {
             result = getParser(codecType).parse(ccContent, 0);
             textTracks.addCaptions(textTracks.getCurrentTrackIdx(), 0, result);
+            if (instance.buffered) {
+                instance.buffered.add(chunk.start, chunk.end);
```

Comment:
why do we need this?

Reply:
This has been added also to support VTT fragmented, for which use case we do want to know buffered range. But not in case of plain sideloaded VTT